test/unit/test_cli.py

```diff
@@ -96,5 +100,82 @@ def test_set_audit_api_url(self, mock_getuser_func):
             response = mocker.last_request.body.decode('utf-8')
             actual_body = json.loads(response)
 
-            self.assertEqual(mocker.call_count, 1)
+            self.assertEqual(mocker.call_count, 2)
             self.assertEqual(expected_body, actual_body)
+
+    @patch('getpass.getuser')
+    @patch('requests.post')
+    def test_post_audit_info_in_progress(self, mock_post, mock_getuser_func):
+        """Test Audit API helper function with no status code"""
+        mock_getuser_func.return_value = 'mockuser'
+        fake_url = 'foo.bar'
+        os.chdir(os.path.normpath(os.path.dirname(__file__) + '/../helpers'))
+
+        _post_audit_info(
+            fake_url,
+            os.path.join(os.getcwd(), 'mock_directory/config/.tf_wrapper'),
+            12345,
+        )
+
+        mock_post.assert_called_once_with(
+            'foo.bar/audit_info',
+            json={
+                'directory': '/test/helpers/mock_directory/config/.tf_wrapper',
+                'start_time': 12345,
+                'status': Status.IN_PROGRESS,
+                'run_by': 'mockuser',
+                'output': None
+            }
+        )
+
+    @patch('getpass.getuser')
+    @patch('requests.post')
+    def test_post_audit_info_success(self, mock_post, mock_getuser_func):
+        """Test Audit API helper function with status code 0"""
+        mock_getuser_func.return_value = 'mockuser'
+        fake_url = 'foo.bar'
+        os.chdir(os.path.normpath(os.path.dirname(__file__) + '/../helpers'))
+
+        _post_audit_info(
+            fake_url,
+            os.path.join(os.getcwd(), 'mock_directory/config/.tf_wrapper'),
+            12345,
+            0
+        )
+
+        mock_post.assert_called_once_with(
+            'foo.bar/audit_info',
+            json={
+                'directory': '/test/helpers/mock_directory/config/.tf_wrapper',
+                'start_time': 12345,
+                'status': Status.SUCCESS,
+                'run_by': 'mockuser',
+                'output': None
+            }
+        )
+
+    @patch('getpass.getuser')
+    @patch('requests.post')
+    def test_post_audit_info_failed(self, mock_post, mock_getuser_func):
+        """Test Audit API helper function with status code 4"""
+        mock_getuser_func.return_value = 'mockuser'
+        fake_url = 'foo.bar'
+        os.chdir(os.path.normpath(os.path.dirname(__file__) + '/../helpers'))
+
+        _post_audit_info(
+            fake_url,
+            os.path.join(os.getcwd(), 'mock_directory/config/.tf_wrapper'),
+            12345,
+            4
+        )
+
+        mock_post.assert_called_once_with(
+            'foo.bar/audit_info',
+            json={
+                'directory': '/test/helpers/mock_directory/config/.tf_wrapper',
+                'start_time': 12345,
+                'status': Status.FAILED,
```

Comment:
nitpick... no need to change if you don't want to, but i see that the only difference b/w the three tests is that the status is different. you could just have a list of test case values (the three diff status vals) then iterate through that in a loop... and reset mock_post after assert